frontend/hooks/player.ts

```diff
@@ -0,0 +1,25 @@
+import UIfx from "uifx";
+import { useState, useEffect, Dispatch, SetStateAction } from "react";
+
+export function usePlayer(
+    audio: string
+): [boolean, Dispatch<SetStateAction<Boolean>>, () => void] {
+    const [player, setPlayer] = useState<UIfx | undefined>();
```

Comment:
I know I used useState to begin with, but could you change this to a ref? It will save one unnecessary render